### DISCUSSION TITLE
how can i connect instagram in angularx

### DISCUSSION THREAD

**Discussion by santhoshkumar-27** [abacritt/angularx-social-login#492] [Q&A] [answered]
how can i  connect instagram in angular give some hope

**Comment by Heatmanofurioso** [comment, marked as answer]
Give a read to this response https://github.com/abacritt/angularx-social-login/issues/133#issuecomment-441414862

TLDR: Unfortunately, you can't use Instagram for this authentication method, at least in the way the library is set up.

I believe you could maybe set it up with a client-sided implicit flow. But the library currently doesn't support any of this
